controllers/component_controller.go

```diff
@@ -528,8 +543,7 @@ func (r *ComponentReconciler) generateGitops(ctx context.Context, req ctrl.Reque
 	}
 	component.Status.GitOps.CommitID = commitID
 
-	// Remove the temp folder that was created
-	return r.AppFS.RemoveAll(tempDir)
```

Comment:
you need to make this call in `GenerateGitopsBase()` otherwise we will have a host of temp dirs for every reconcile

Reply:
Fixed